src/Compilers/VisualBasic/Portable/BoundTree/BoundNodes.xml

```diff
@@ -553,7 +553,7 @@
     </Node>
 
     <!-- Use this in the event that the sequence point must be applied to expression.-->
-    <Node Name="BoundSequencePointExpression" Base="BoundExpression">
+    <Node Name="BoundSequencePointExpression" Base="BoundExpression" HasValidate="true">
```

Comment:
@333fred @jcouv @cston This is what causing the CI failure. This is what I see in the dump

![image](https://user-images.githubusercontent.com/31348972/170763004-cf908c85-eb64-4e44-a68a-5d1cfbd1375c.png)

The CompileMethod call in the stack trace is attempting to compile this method:

https://github.com/dotnet/roslyn/blob/9c4920a8d2538abd92d7e8880f556f75ad8a6588/src/Compilers/Test/Utilities/VisualBasic/CompilationTestUtils.vb#L285-L297

What happens is:

https://github.com/dotnet/roslyn/blob/9c4920a8d2538abd92d7e8880f556f75ad8a6588/src/Compilers/VisualBasic/Portable/Generated/BoundNodes.xml.Generated.vb#L12287-L12291

`node.Update` is expecting `expression.Type` and `type` to be the same. `type` is originating from here:

https://github.com/dotnet/roslyn/blob/9c4920a8d2538abd92d7e8880f556f75ad8a6588/src/Compilers/VisualBasic/Portable/Lowering/MethodToClassRewriter/MethodToClassRewriter.vb#L97-L103

I'm unable to investigate that further :(